Lombiq.HelpfulExtensions.csproj

```diff
@@ -35,4 +35,8 @@
       Condition="Exists('..\..\Libraries\Lombiq.HelpfulLibraries\Lombiq.HelpfulLibraries.csproj')" />
   </ItemGroup>
 
+  <ItemGroup Condition="'$(NuGetBuild)' == 'True'">
+    <PackageReference Include="Lombiq.HelpfulLibraries.Targets" Version="0.0.2" />

```

Comment:
this would want to be updated every time HL is published.

(but referencing HL directly in the consuming project, would probably be enough, to force it up a version, if there were no breaking changes in HL)